### PR TITLE
Feature/cookies preferences page

### PR DESCRIPTION
**Pull request by jondewijones**
### What

Added cookies preference form page

### How to review

Pull https://github.com/ONSdigital/dp-frontend-cookie-controller
Pull https://github.com/ONSdigital/sixteens/tree/feature/radio-styling

Test page by going to `http://localhost/cookies/edit`

### Who can review

Anyone

### CODE REVIEW

File: assets/templates/cookies-preferences.tmpl [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    <h3 class="font-weight-700 margin-top--2">{{ localise "CookiesEssential" .Language 1}}</h3>
+
+    <p>{{ localise "CookiesEssentialP1" .Language 1}}</p>
+
+    <p>{{ localise "CookiesEssentialP2" .Language 1}}</p>
+    
+    <p>{{ localise "CookiesEssentialP3" .Language 1 | safeHTML }}</p>
+
+    <button class="btn btn--primary btn--thick btn--wide btn--big btn--focus margin-top--2 margin-right--2 font-weight-700 font-size--17 text-wrap" type="submit">{{ localise "SaveChanges" .Language 2}}</button>
```
Comment (rav-pradhan) [inline assets/templates/cookies-preferences.tmpl]: Should we indent lines 57 - 65 while they're within the form element for markup consistency?

File: handlers/cookies/handler.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+	"github.com/ONSdigital/dp-frontend-models/model"
+	"github.com/ONSdigital/dp-frontend-renderer/render"
+)
+
+//Handler ...
+func Handler(w http.ResponseWriter, req *http.Request) {
+	var page model.Page
+
+	fmt.Printf("%+v\n", page)
```
Comment (rav-pradhan) [inline handlers/cookies/handler.go]: If this isn't necessary anymore for debugging etc. then we ought to remove this

File: assets/templates/cookies-preferences.tmpl [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            <div>
+                <div class="radio margin-top--1 radio--inline">
+                    <input class="radio__input"  type="radio" name="cookie-policy-usage" id="usage-on" {{ if eq .CookiesPolicy.Usage true }}checked{{ end }}/>
+                    <label class="radio__label" for="usage-on">{{ localise "On" .Language 1}}</label>
+                </div>
+                <div class="radio margin-top--1 radio--inline">
+                    <input class="radio__input"  type="radio" name="cookie-policy-usage" id="usage-off" {{ if eq .CookiesPolicy.Usage false }}checked{{ end }} />
+                    <label class="radio__label" for="usage-off">{{ localise "Off" .Language 1}}</label>
+                </div>
+            </duv>
```
[the remark this answers is not in the document]
Comment (jondewijones) [reply assets/templates/cookies-preferences.tmpl]: 👍

File: assets/templates/cookies-preferences.tmpl (hunk 1 above)
Comment (jondewijones) [reply assets/templates/cookies-preferences.tmpl]: 👍

File: handlers/cookies/handler.go (hunk 2 above)
Comment (jondewijones) [reply handlers/cookies/handler.go]: 👍

**Pull request merged**